packages/terra-clinical-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import memoize from 'memoize-one';
+
+import { KEYCODES } from '../utils/keycodes';
+
+import styles from './Cell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String identifier of the section in which the Cell will be rendered.
+   */
+  sectionId: PropTypes.string.isRequired,
+  /**
+   * String identifer of the row in which the Cell will be rendered.
+   */
+  rowId: PropTypes.string.isRequired,
+  /**
+   * String identifier of the column in which the Cell will be rendered.
+   */
+  columnId: PropTypes.string.isRequired,
+  /**
+   * String-formatted width that the Cell should be rendered as. Any valid css width value is supported (i.e. 200px, 3rem).
+   */
+  width: PropTypes.string.isRequired,
+  /**
+   * Boolean indicating whether the Cell is selectable.
+   */
+  isSelectable: PropTypes.bool,
+  /**
+   * Boolean indicating whether the Cell is actively selected.
+   */
+  isSelected: PropTypes.bool,
+  /**
+   * Function that will be called upon Cell selection. The `isSelectable` prop must be true for this function to be called.
+   */
+  onSelect: PropTypes.func,
+  /**
+   * Function that will be called upon the mouse entering the selectable region of the Cell. The `isSelectable` prop must be true for this function to be called.
+   */
+  onHoverStart: PropTypes.func,
+  /**
+   * Function that will be called upon the mouse leaving the selectable region of the Cell. The `isSelectable` prop must be true for this function to be called.
+   */
+  onHoverEnd: PropTypes.func,
+  /**
+   * Content that will rendered within the Cell.
+   */
+  children: PropTypes.node,
+  /**
+   * Function that will be called with a ref to the Cell's selectable element.
+   */
+  selectableRefCallback: PropTypes.func,
+};
```

Comment:
😞I dug back through the war room notes and the only thing I found (from March of this year) said:

>When the prop is a primitive such as a boolean, string, or number, don't worry about specifying a default value. Leave it as undefined.

Though I feel like we've re-litigated that since then.

Reply:
hmmmmmm okay. That requires three checks then instead of two for true, false and undefined?

Reply:
If I'm correct, the original need for it came up to address the need for the selectable tables for when the component was first initialized without user selection, but wanted styles for when users selected and deselected values in the list.

I think undefined should be ok in this scenario.